### ISSUE TITLE
Cluster status turn to red when close partition

### ISSUE THREAD

**Issue by DLT1412** [crate/crate#6488]
**CrateDB version**:
2.1.8
**Problem description**:
Cluster status turn to red when close partition of table.
**Steps to reproduce**:
```
create table p (a integer) partitioned by (a);
insert into p values (1),(2);
alter table p partition (a=1) close;
```

**Comment by joemoe** [comment]
hi @DLT1412, thanks. we will look into that.

**Comment by m-kharbat** [comment]
this issue will be fixed in 2.1.10
